### ISSUE TITLE
Don't use the wrapper

### ISSUE THREAD

**Issue by TNovalis** [DaaOtt/maestro#66]
The wrapper is bad..

```
[mysql] MySQL Query Error!
Query: INSERT INTO `maestro_ranks` (`rank`, `inherits`, `cantarget`, `canrank`, `color`) VALUES ("admin", "user", "<^", "!>^", "255 255 255")
Duplicate entry 'admin' for key 'PRIMARY'
[mysql] MySQL Query Error!
Query: INSERT INTO `maestro_ranks` (`rank`, `inherits`, `cantarget`, `canrank`, `color`) VALUES ("superadmin", "admin", "<^", "!>^", "255 255 255")
Duplicate entry 'superadmin' for key 'PRIMARY'
[mysql] MySQL Query Error!
Query: INSERT INTO `maestro_ranks` (`rank`, `inherits`, `cantarget`, `canrank`, `color`) VALUES ("user", "user", "<^", "!>^", "255 255 255")
Duplicate entry 'user' for key 'PRIMARY'
```

..because it doesn't support the ON DUPLICATE KEY.

Just rewrite your mysql and only have it be [mysqloo](https://facepunch.com/showthread.php?t=1357773) (the most commonly used).

Also use conditional logic to know if you run the query or not... I swear.

**Comment by andreblue** [comment]
mysqloo sucks........ tmysql is much better and does not have memory leaks

**Comment by ottworks** [comment]
That error was harmless (and actually intended behavior). CAMI was adding ranks before I had loaded mine--I've fixed it in 2.0.3
